src/Maker/MakeResetPassword.php

```diff
@@ -186,22 +198,51 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'Form\\'
         );
 
+        /*
+         * @legacy Conditional can be removed when MakerBundle no longer
+         *         supports Symfony < 6.0.
```

Comment:
Same here - 5.3 is the min version when we can drop this.

Reply:
done